.github/workflows/build.yml

```diff
@@ -228,8 +228,8 @@ jobs:
 
       - name: Test CosmosDB
         env:
-          MT_COSMOS_ENDPOINT: ${{ secrets.AZURE_COSMOSENDPOINT }}
-          MT_COSMOS_KEY: ${{ secrets.AZURE_COSMOSKEY }}
+          MT_COSMOS_ACCOUNT_ENDPOINT: ${{ secrets.AZURE_COSMOSENDPOINT }}
```

Comment:
And changing the environment variable breaking my local tests. I'd just suggest putting these back to what they were.

Reply:
Reverted the names for the env variables.